server/src/controller/apis/README.md

```diff
@@ -4,5 +4,20 @@
 
 - Health API: Request -> GET: `/api/v1/user/health` . 
 
-Expected Responses:
-1. Response: status= 200 and body = `{status: 'Ok'}` 
+    Expected Responses:
+    1. User API reachable => Response: status= 200 and body = `{status: 'Ok'}` 
+    2. User API not reachable => Response: No response or Request timeout.
```

Comment:
Let's standardize timeout to be used in frontend.

Reply:
Yes. Let's do it in the next team meeting. :smile: @nihalanisumit